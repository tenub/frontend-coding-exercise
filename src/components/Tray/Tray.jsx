import React from 'react';

const Tray = ({ children }) => (
  <div className='element-tray'>
    {children}
  </div>
);

export default Tray;
