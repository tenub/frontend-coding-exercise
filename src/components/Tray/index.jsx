import Tray from './Tray';

export default Tray;
