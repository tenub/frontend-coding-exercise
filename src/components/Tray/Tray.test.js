import React from 'react';
import TestRenderer from 'react-test-renderer';

import Tray from './';

test('Tray renders successfully', () => {
  const tray = (
    <Tray>
      Child text
    </Tray>
  );
  const testRenderer = TestRenderer.create(tray);
  const testTree = testRenderer.toJSON();
  expect(testTree).toMatchSnapshot();
});
