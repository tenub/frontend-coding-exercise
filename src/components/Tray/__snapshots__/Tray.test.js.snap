// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tray renders successfully 1`] = `
<div
  className="element-tray"
>
  Child text
</div>
`;
